scipy/stats/tests/test_stats.py

```diff
@@ -4200,6 +4212,105 @@ def test_kurtosistest_too_few_samples():
     assert_raises(ValueError, stats.kurtosistest, x)
 
 
+class TestMann_Whitney_U():
```

Comment:
These tests were added in the original PR for `mann_whitney_u`. These will be removed, and all the ones we want to keep will be in `test_hypotests.py`.